src/actions/rating.action.js

```diff
@@ -0,0 +1,75 @@
+import CONSTANTS from '../constants/index';
+import api from '../api';
+
+const { RATE_ARTICLE, AVERAGE_RATING, RATING_ERROR } = CONSTANTS;
+export const rateArticle = data => ({
+  /**
```

Comment:
Block comments in javascript are placed above the function.
Refer here `http://usejsdoc.org/about-getting-started.html`